src/ai-dialog-footer.ts

```diff
@@ -1,44 +1,46 @@
+import {inject} from 'aurelia-dependency-injection';
 import {customElement, bindable, inlineView} from 'aurelia-templating';
 import {DialogController} from './dialog-controller';
 
 /**
- * * View-model for footer of Dialog.
- * */
+ * View-model for footer of Dialog.
+ */
 @customElement('ai-dialog-footer')
+/* tslint:disable:max-line-length */
 @inlineView(`
   <template>
     <slot></slot>
-
     <template if.bind="buttons.length > 0">
       <button type="button" class="btn btn-default" repeat.for="button of buttons" click.trigger="close(button)">\${button}</button>
```

Comment:
We can make this - 

```
<button type="button" class="btn btn-default"
  repeat.for="button of buttons"
  click.trigger="close(button)">\${button}</button>
```
To comply to the rule.  I'd rather not have files disabling linting in them unless we absolutely need to (such as copying over some open source file and not wanting to modify the source)